media-gfx/slic3r/slic3r-1.3.0.ebuild

```diff
@@ -0,0 +1,115 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eutils perl-module
```

Comment:
`make_desktop_entry` is currently provided by the `desctop.eclass`, so replace eutils